src/Microsoft.Diagnostics.Runtime.Tests/Targets/GCHandles.cs

```diff
@@ -13,11 +14,22 @@ public static void Main()
         string weak = "weak";
         GCHandle.Alloc(weak, GCHandleType.Weak);
 
+        //var a = new System.Net.Sockets.SocketAsyncEventArgs();
```

Comment:
This shouldn't make it to the final change.

Reply:
My bad, went too fast. Fixed.